PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutConfig.java

```diff
@@ -0,0 +1,65 @@
+package com.braintreepayments.api;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+public class PayPalNativeCheckoutConfig implements Parcelable {
```

Comment:
I just added this to be safe in case we wanted to support adding in the 3p callbacks here later. I believe both of these things could be moved or fetched elsewhere as well

Reply:
Can we remove this for now and add later?

Reply:
yeah we can remove this, we can use the `PayPalNativeCheckoutRequest` to hold these for now